src/test/java/core/basesyntax/service/FruitTransactionParserTest.java

```diff
@@ -0,0 +1,69 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import org.junit.jupiter.api.Test;
+
+class FruitTransactionParserTest {
```

Comment:
I don`t see a positive cases here

Reply:
it was commented test which idk how to fix